preprocess.py

```diff
@@ -0,0 +1,91 @@
+'''
+Crated on June 3, 2020
+Author: Joe Tawea
+
+To Do:
+    try-except to check folder before writing
+'''
+
+import sys
+import pandas as pd
+
+# Import Review-Data
+# Remove duplicate reviewerID and asin since book format (hardcover, paperback)
+# Convert reviewerID and asin to categorical
+# Save the DataFrame to pkl
+
+def preprocess_review(review_file, review_pkl):
+
+    print(f'  Reading {review_file}')
+    try:
+        review_df = pd.read_csv(review_file,
+                                dtype={'asin': object, 'reviewerID':object},
+                                sep=',')
+    except:
+        print(f'Error loading {review_file}\nExit!')
+        sys.exit(1)
+
+    print(f'  Done reading {review_file}')
+
+    review_df.drop_duplicates(subset=['reviewerID','asin'],
+                              keep='first', inplace=True)
```

Comment:
this kind of formatting is totally fine! But you seem like you care about your code looking uniform and clean (which is really good). I just wanted to point out this tool that I think you might like: 
https://black.readthedocs.io/en/stable/

The typical use case would be to run the `black` linter on your whole directory before you push a commit. If you use VSCode, you can even set it to automatically trigger everytime you save a file.